lua/obsidian/command.lua

```diff
@@ -330,32 +330,59 @@ command.complete_args = function(client, _, cmd_line, _)
   return completions
 end
 
----Follow link under cursor.
+---Find backlinks for the note under the cursor.
 ---
 ---@param client obsidian.Client
-command.follow = function(client, _)
-  local scan = require "plenary.scandir"
-
-  local open, close = util.cursor_on_markdown_link()
-  local current_line = vim.api.nvim_get_current_line()
-
-  if open == nil or close == nil then
-    echo.err "Cursor is not on a reference!"
+command.cursor_backlinks = function(client, _)
+  local note_name, _ = util.get_note_name()
```

Comment:
I think we want the note file name here, not the note name.
```suggestion
  local _, note_file_name = util.get_note_name()
```